index.js

```diff
@@ -29,7 +33,7 @@ const DESCRIPTION_MAP = {
 * @return {Object}             An object with the user, repo, and branch
 */
 function getInfo(scmUrl) {
-    const matched = (schema.config.regex.SCM_URL).exec(scmUrl);
+    const matched = (schema.config.regex.CHECKOUT_URL).exec(scmUrl);
```

Comment:
Opening a pull request to fix the regex for `CHECKOUT_URL`
https://github.com/screwdriver-cd/data-schema/pull/70/files#diff-77dea1a61b658aff6ad9cd0d2528d2daR18
Might want to double check your matching logic